### PR TITLE
Lighthouse Fixes

### LINKED ISSUE #94: Improve Lighthouse Reports
[**Repldex - Lighthouse Report**](https://lighthouse-dot-webdotdevsite.appspot.com//lh/html?url=https%3A%2F%2Frepldex.com)

I'm gonna be working on improving the scores for a lot of the failed tests in the Lighthouse Report, specifically in `Accessibility` and `Performance`. This is all frontend stuff so it shouldn't interfere with what y'all have going on with the DB, bot, server, etc.

### PR DESCRIPTION
**Pull request by IreTheKID**
Closes #94

### CODE REVIEW

**Comment by IreTheKID** [comment]
@mat-1 @Codemonkey51 this PR is ready to be merged, let me know if you want me to change or add smth.

**Comment by Codemonkey51** [comment]
> @mat-1 @Codemonkey51 this PR is ready to be merged, let me know if you want me to change or add smth.

Looks good, but someone should test it I'll request that rn

**Comment by IreTheKID** [comment]
OK, sure

File: repldex/backend/templates/entry.html [hunk 1]
```diff
@@ -8,87 +8,60 @@
...
-  <option value="1" selected>English</option>
-		<option value="2">Lolcat</option>
-		<option value="3">Pirate</option>
-    <option value="4">Morse Code</option>
-    <option value="5">Shakespeare</option>
-    <option value="6">Gungan</option>
-    <option value="7">Leet</option>
-    <option value="8">Pig Latin</option>
-}
-
```
Comment (mat-1) [inline repldex/backend/templates/entry.html:90]: Removal of TTS and translations should be in a different PR

File: repldex/backend/templates/entry.html [hunk 2]
```diff
@@ -8,87 +8,60 @@
 		<meta property="og:image" content="{{ image }}">
 	{% endif %}
 
-	<script>
+<script>
 document.addEventListener('DOMContentLoaded', () => {
```
Comment (mat-1) [inline repldex/backend/templates/entry.html:12]: You converted all my beautiful tabs into ugly spaces >:(

File: repldex/backend/templates/base.html [hunk 3]
```diff
@@ -1,53 +1,33 @@
...
+<html lang="en">
+
 <head>
 {% block head %}
 <title>{% block title %}Repldex{% endblock %}</title>
+
 <link rel="icon" type="image/png" href="/static/icon.png">
-<link rel="stylesheet" href="/static/main.css">
+<link rel="preload" href="/static/main.css" as="style">
+<link rel="preload" href="/static/lazy-images.js" as="script">
```
Comment (mat-1) [inline repldex/backend/templates/base.html:10]: Why move this to a different file?

File: repldex/backend/static/main.css [hunk 4]
```diff
@@ -1,4 +1,4 @@
...
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap');
```
Comment (mat-1) [inline repldex/backend/static/main.css:1]: Why switch to Google fonts? We could even put the font as a static file in Repldex so it doesn't have to preconnect.

File: repldex/backend/templates/base.html [hunk 5]
```diff
@@ -1,53 +1,33 @@
...
+
```
Comment (mat-1) [inline repldex/backend/templates/base.html:11]: You should be preconnecting to i.matdoes.dev

File: repldex/backend/templates/index.html [hunk 6]
```diff
@@ -141,7 +141,10 @@ <h1 class="title">Repldex</h1>
 			{% endif %}
     {% endif %}
   </div>
-	<input id="search-bar" placeholder="Search entries">
+    <label>
+        Search entries
```
Comment (mat-1) [inline repldex/backend/templates/index.html:145]: This should be an aria label or something so it doesn't show the text normally

**Review by mat-1 — CHANGES_REQUESTED** [review]

File: repldex/backend/static/main.css (hunk 4 above)
Comment (IreTheKID) [reply repldex/backend/static/main.css:1]: I switched to Google Fonts because you guys put way too many requests to the other URL

File: repldex/backend/templates/index.html (hunk 6 above)
Comment (IreTheKID) [reply repldex/backend/templates/index.html:145]: Sure

File: repldex/backend/templates/base.html (hunk 5 above)
Comment (IreTheKID) [reply repldex/backend/templates/base.html:13]: Sure

File: repldex/backend/templates/base.html (hunk 3 above)
Comment (IreTheKID) [reply repldex/backend/templates/base.html:12]: It's used in almost every route, I thought preloading it would be better than inline JS for every page.

File: repldex/backend/templates/entry.html (hunk 2 above)
Comment (IreTheKID) [reply repldex/backend/templates/entry.html:12]: no shush

File: repldex/backend/templates/entry.html (hunk 1 above)
Comment (IreTheKID) [reply repldex/backend/templates/entry.html:90]: It corresponds to a Lighthouse issue, so idrc

Comment (jetstream0) [reply repldex/backend/templates/entry.html:90]: yeah seems fair

**Comment by IreTheKID** [comment]
@mat-1 re-review?

**Review by mat-1 — CHANGES_REQUESTED** [review]
fix the indentation first smh. (Tabs save 3 bytes over spaces too, which makes the page load nanoseconds faster to increase our lighthouse score!)

**Comment by IreTheKID** [comment]
ugh, fineeee

**Comment by IreTheKID** [comment]
@mat-1 happy now?

**Review by mat-1 — APPROVED** [review]

**Review by jetstream0 — COMMENTED** [review]
Yeah everything seems fine
